### PR TITLE
Populate the template with initial content

### PR DESCRIPTION
**Pull request by ananos**
Add content from the docs: https://docs.google.com/document/d/1Q-Od2pFIE73JGnLdCShN1-EfiLO9VoV6ejaR_8_7BwI/edit

Update the front page content (default layout, seems a bit rough -- needs content + polishing).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
 The aim is to promote unikernel technologies in commercially viable products, in research and in education.
 
 This repository stores information for the [Unikernel Alliance site](https://unikernelalliance.org).
+
+### Build locally
```
Comment (razvand) [inline README.md:8]:
```suggestion
## Build locally
```

File: README.md [hunk 2]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+
+#### Clone the repo:
```
Comment (razvand) [inline README.md:10]:
```suggestion
### Clone the repo:
```

File: README.md [hunk 3]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+
+```
+git clone https://github.com/unikernelalliance/unikernelalliance.org
+```
+
+#### Fetch dependencies:
```
Comment (razvand) [inline README.md:16]:
```suggestion
### Fetch dependencies:
```

File: README.md [hunk 4]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+
+```
+cd unikernelalliance.org
+npm install
+```
+
+#### (a) Run locally:
```
Comment (razvand) [inline README.md:23]:
```suggestion
### (a) Run locally:
```

File: README.md [hunk 5]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+  Static files     | 72  
+  Processed images | 16  
+  Aliases          |  5  
+  Sitemaps         |  2  
+  Cleaned          |  0  
+
+Total in 1150 ms
+```
+
+#### Auto-deploy on github pages:
```
Comment (razvand) [inline README.md:192]:
```suggestion
## Auto-deploy on GitHub Pages
```

File: README.md [hunk 6]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+
+There's a deploy action through Github Actions in
+`.github/workflows/deploy-github.yml`. Essentially, this action builds the
+static website and adds the `public/` dir to a separate branch `gh-pages`.
+Enabling the `Pages` functionality of Github, allows the website to be deployed
+to `unikernelalliance.github.io` or to a custom domain we set from the
+`Settings`->`Pages`->`custom domain` option of the repository.
```
Comment (razvand) [inline README.md:199]:
```suggestion
There's a deploy action through Github Actions in `.github/workflows/deploy-github.yml`.
Essentially, this action builds the static website and adds the `public/` dir to a separate branch `gh-pages`.
Enabling the `Pages` functionality of Github allows the website to be deployed to `unikernelalliance.github.io` or to a custom domain we set from the `Settings`->`Pages`->`custom domain` option of the repository.
```

Use one row per sentence.

File: README.md [hunk 7]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+
+
```
Comment (razvand) [inline README.md:201]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
...
+**Note**: The above was derived from: https://github.com/h-enk/doks-gh-pages, https://getdoks.org/docs/start-here/getting-started/ and we had to manually add `packages-lock.json`
```
Comment (razvand) [inline README.md:202]:
```suggestion
**Note**: The above was derived from: https://github.com/h-enk/doks-gh-pages, https://getdoks.org/docs/start-here/getting-started/.
We manually added `packages-lock.json`
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @ananos . This looks good. Please see my comments. Please use full sentence, start with capital letters for commit messages. I.e. use:

```text
content: Add initial content

Update projects and front layout. This is a rough first version, aimed to be improved.
```

Instead of:

```text
content: add initial content

update projects, and front layout (rough)
```

File: README.md (hunk 5 above)
Comment (ananos) [reply README.md]: TODO: capitalize github pages, will do that on the 2nd iteration of pushes.

**Review by razvand — APPROVED** [review]
All good. Thanks, @ananos

**Pull request merged**